plugins/USBPrinting/USBPrinterOutputDeviceManager.py

```diff
@@ -83,7 +83,7 @@ def _updateThread(self):
             if container_stack.getMetaDataEntry("supports_usb_connection"):
                 machine_file_formats = [file_type.strip() for file_type in container_stack.getMetaDataEntry("file_formats").split(";")]
                 if "text/x-gcode" in machine_file_formats:
-                    port_list = self.getSerialPortList(only_list_usb=True)
+                    port_list = self.getSerialPortList(only_list_usb=False)
```

Comment:
FieldOfView made a good point about this annoying users without this issue. I think it would be a good idea to only set this to false when Cura is running on MacOS. You can see an example of this here

[https://github.com/Ultimaker/Uranium/blob/41e98b8686911384ef5088dac7b8d15e52e33cb1/UM/View/GL/OpenGLContext.py#L146](https://github.com/Ultimaker/Uranium/blob/41e98b8686911384ef5088dac7b8d15e52e33cb1/UM/View/GL/OpenGLContext.py#L146)

Also it would be good to add a comment that mentions the issue, so we can change this back in the future if it is fixed.